pkg/sql/codegen/pai/template_tf.go

```diff
@@ -56,8 +57,16 @@ assert driver == "maxcompute"
 user, passwd, address, database = sqlflow_submitter.db.parseMaxComputeDSN(dsn)
 
 jobname = '_'.join(['sqlflow', '{{.ModelName}}'.replace('.', '_')])
-pai_cmd = 'pai -name %s -DjobName=%s -Dtags=%s -Dscript=file://%s -DentryFile=%s -DgpuRequired=\'\' -Dtables=odps://%s/tables/%s' % (
+# The tags candidate list is: "cnn,dnn,rnn,bert,ctr,cvr,inception,resnet,gnn,gcn,ocr,maskrcnn,transformer,nmt,others". Use "others" if you are not sure which tags you need.
+# Need to add arguments -Dbuckets="oss://..." -DcheckpointDir="oss://..." for distributed training.
+{{if gt .NumWorkers 1}}
```

Comment:
I saw `codegen.go#122` to make the default value of `NumPS`  is `0`, do we need the `NumPS` here?

Reply:
`numWorkers > 1` is what we need to determine whether this is a distributed job